bucket/regexp.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "0.7",
+    "description": "Registry Explorer - enhanced Registry editor/viewer",
+    "homepage": "https://github.com/zodiacon/RegExp",
+    "license": "MIT",
+    "url": "https://github.com/zodiacon/RegExp/releases/download/v0.7/RegExp.exe",
+    "hash": "e4882b728070988c222ec23d78fb0bc6184c6f1757613b2bb90ec05afc51d4d0",
+    "bin": "RegExp.exe",
```

Comment:
Command line does not work
![image](https://user-images.githubusercontent.com/13260377/127763828-1c18eab6-a3d0-4efe-9176-ee8ac64c941a.png)

```suggestion
```